app/main.ipynb

```diff
@@ -192,18 +244,27 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "means = np.mean(A, axis=0)\n",
+    "print(means)"
```

Comment:
The calculation of the mean for each column using `np.mean` with `axis=0` is correct and matches the task requirement.